rre-search-platform/rre-search-platform-elastic-search-impl/src/main/java/io/sease/rre/search/api/impl/Elasticsearch.java

```diff
@@ -199,7 +201,12 @@ public QueryOrSearchResponse executeQuery(final String indexName, final String q
                                 return result;
                             })
                             .collect(toList()));
-        } catch (final IOException exception) {
+        }
+        catch (ElasticsearchException e) {
```

Comment:
Just a minor thing @mattoscln @mattflax could you please put the catch in the same line of }?
I know, that's a matter of style, no good or bad, but I think we should follow the same style as much as possible ok?

Reply:
Thanks for this - I should have checked the formatting before committing. I've pushed changes to this and the other formatting issues.